examples/webgl_loader_sea3d_hierarchy.html

```diff
@@ -120,6 +116,9 @@
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.setClearColor( 0x333333, 1 );
+				renderer.shadowMap.enabled = true;
+				renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+				renderer.shadowMap.cullFace = THREE.CullFaceBack;
```

Comment:
@tschw About the `Mixer` you know if it is possible to change the `timeScale` from action: